### DISCUSSION TITLE
Why does PK not change after setting DDI？

### DISCUSSION THREAD

**Discussion by noxavier** [Open-Systems-Pharmacology/Forum#1256] [Questions & Problems] [answered]
PK SIM simulates drug interactions only through transporters to affect drug-drug interaction. Why does PK not change after setting? What might go wrong?
Thank you in advance :)

![微信截图_20220721111647](https://user-images.githubusercontent.com/109507330/180122500-52f1c5f9-a3f1-4719-904c-c5862a72aae5.jpg)

**Comment by DeniTue** [comment, marked as answer]
Dear @noxavier,

Your observation can have different causes. First, please make sure that the Ki is selected when configuring the DDI simulation (under "Processes") and is linked to the correct transporter. If this is the case, it could be possible that the interaction is "too weak" because, e.g., the concentration of the perpetrator on the target is too low or the Ki value is too large. You can try to simulate the unbound concentration of your perpetrator in plasma or in transporter-relevant tissues. Also, the victim PK might not be sensitive enough to the transport process and therefore inhibition has only little effect.

Best,
Denise

**Reply by noxavier** [reply]
Thank you very much！
